src/averageAges.js

```diff
@@ -59,6 +82,37 @@ function calculateAverageAgeDiff(people, onlyWithSon) {
   // 2. keep people who have mothers in the array
   // 3. calculate the difference child.born - mother.born
   // 4. return the average value
+  const peopleWithMother = onlyWithSon
+    ? people.filter((person) =>
+      person.mother !== null && person.sex === 'm'
```

Comment:
Consider using constants with meaningful names instead of magic strings and numbers it's a considered anti-pattern